src/main/java/fr/quatrevieux/araknemu/game/GameConfiguration.java

```diff
@@ -73,6 +74,14 @@ public String ip() {
         return pool.string("server.ip", "127.0.0.1");
     }
 
+    /**
+     * Get the configured time zone.
+     * Default: system default time zone
+     */
+    public ZoneId timezone() {
+        return pool.zoneId("server.timezone", ZoneId.systemDefault());
+    }
```

Comment:
Add the unit test also for this method on `GameConfigurationTest`.

Reply:
Fixed in commit [#89d9b49](https://github.com/Arakne/Araknemu/pull/309/commits/89d9b49917835fcc9e2ee816f3221f1efe7ef7d4)